packages/hash/frontend/src/components/hooks/blockProtocolFunctions/useBlockProtocolUpdateLinkedAggregations.ts

```diff
@@ -0,0 +1,66 @@
+import { useMutation } from "@apollo/client";
+
+import {
+  BlockProtocolLinkedAggregationUpdateMutationResults,
+  BlockProtocolUpdateLinkedAggregationsFunction,
+} from "blockprotocol";
+
+import { updateLinkedAggregationMutation } from "@hashintel/hash-shared/queries/link.queries";
+import { useCallback } from "react";
+import {
+  UpdateLinkedAggregationOperationMutation,
+  UpdateLinkedAggregationOperationMutationVariables,
+} from "../../../graphql/apiTypes.gen";
+
+export const useBlockProtocolUpdateLinkedAggregations = (
+  /** Providing accountId here saves blocks from having to know it */
+  sourceAccountId: string,
+): {
```

Comment:
There is no `useBlockProtocolUpdateLinkedAggregations` in the block protocol - please call this `useBlockProtocolUpdateLinks` instead, and call the `updatedLinkedAggregations` mutation if the link action has an `"operation"` key

Reply:
Cool, created this function with a `@todo` to implement the linkedEntities/linkGroups updating part in the future: https://github.com/hashintel/hash/pull/188/files#diff-ed1c3db2c08507a513393a3fe91d0198d06855d4afbbdd34b2414f0f3b6f593dR27